.env

```diff
@@ -0,0 +1 @@
+REACT_APP_FIREBASE_KEY= AIzaSyC08_5PeIG320NtW2dBSvaXy7A3yV6nlHA
```

Comment:
Take care to not push your env keys to public repositories, add it to gitignore

Reply:
Yes I did it by mistake , I will keep in mind from now on